nixos/modules/services/hardware/undervolt.nix

```diff
@@ -161,6 +173,11 @@ in
   };
 
   config = mkIf cfg.enable {
+    assertions = [ {
+      assertion = !hasCliArgs;
+      message = "At least one undervolt parameter must be set to enable service.undervolt";
```

Comment:
This message is insufficient; we should specify which flags are needed for this config to be considered valid.